Console/Command/CakePHPUtilShell.php

```diff
@@ -0,0 +1,330 @@
+<?php
+App::uses('AppShellErrorHandler', 'Lib/Util');
+App::uses('Shell', 'Console');
+
+/**
+ * Application Shell
+ *
+ * Add your application-wide methods in the class below, your shells
+ * will inherit them.
+ *
+ * @package       app.Console.Command
+ *
+ * @property MailTask $Mail
+ */
+class CakePHPUtilShell extends Shell {
+
+	const SUCCESS_MESSAGE = 'Shell action ran successfully and without problems.';
+	/**
+	 * Tasks to use
+	 *
+	 * @var array
+	 */
+	public $tasks = array('Mail');
+
+	/**
+	 * Contains the debug value
+	 *
+	 * @var int
+	 */
+	protected $_debug;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "$_debug" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)